mark-walle-victoria.md

```diff
@@ -0,0 +1,11 @@
+# Victoria, BC
+
+### Food Recommendations
+- Big Wheel Burger
+- Tacofino
+- Standard Pizza
+
+### Places to Go
+- Gonzales Hill Observatory
+- Ogden Point Breakwater
```

Comment:
It's a a man made spit made to protect the Victoria harbor from any strong currents or waves that might pass by.